applications/tari_base_node/osx-pkg/build_pkg.sh

```diff
@@ -39,50 +39,74 @@ else
   exit 2
 fi
 
-mkdir -p "$destDir/pkgRoot"
-mkdir -p "$destDir/pkgRoot/usr/local/bin/"
+mkdir -p "${destDir}/pkgRoot"
+mkdir -p "${destDir}/pkgRoot/usr/local/bin/runtime"
+mkdir -p "${destDir}/pkgRoot/usr/local/bin/config"
 
-COPY_BIN_FILES=(
+VERIFY_BIN_FILES=(
   "tari_base_node"
   "tari_console_wallet"
   "tari_merge_mining_proxy"
   "tari_mining_node"
 )
-for COPY_BIN_FILE in "${COPY_BIN_FILES[@]}"; do
+for VERIFY_BIN_FILE in "${VERIFY_BIN_FILES[@]}"; do
   # Verify signed?
-  codesign --verify --deep --display --verbose=4 "$destDir/dist/$COPY_BIN_FILE"
-  spctl -vvv --assess --type exec "$destDir/dist/$COPY_BIN_FILE"
-
-  cp "$destDir/dist/$COPY_BIN_FILE" "$destDir/pkgRoot/usr/local/bin/"
-done
-
-mkdir -p "$destDir/pkgRoot/usr/local/share/$instName"
-COPY_SHARE_FILES=(
-  *.sh
-)
-for COPY_SHARE_FILE in "${COPY_SHARE_FILES[@]}"; do
-  cp "$destDir/dist/"$COPY_SHARE_FILE "$destDir/pkgRoot/usr/local/share/$instName/"
+  codesign --verify --deep --display --verbose=4 "${destDir}/runtime/$VERIFY_BIN_FILE"
+  spctl -vvv --assess --type exec "${destDir}/runtime/$VERIFY_BIN_FILE"
 done
 
-mkdir -p "$destDir/pkgRoot/usr/local/share/doc/$instName"
-COPY_DOC_FILES=(
-  "tari_sample.toml"
-  "tari_config_example.toml"
-  "log4rs_sample_base_node.yml"
-  "README.md"
-)
-for COPY_DOC_FILE in "${COPY_DOC_FILES[@]}"; do
-  cp "$destDir/dist/"$COPY_DOC_FILE "$destDir/pkgRoot/usr/local/share/doc/$instName/"
-done
-
-mkdir -p "$destDir/scripts"
-cp -r "${sPath}/scripts/"* "$destDir/scripts/"
-
-pkgbuildResult=$(pkgbuild --root $destDir/pkgRoot \
+# One click miner
+cp "${destDir}/start_all" "${destDir}/pkgRoot/usr/local/bin/"
+cp "${destDir}/runtime/start_all.sh" "${destDir}/pkgRoot/usr/local/bin/runtime/"
+
+# Base Node
+cp "${destDir}/start_tari_base_node" "${destDir}/pkgRoot/usr/local/bin/"
+cp "${destDir}/start_tor" "${destDir}/pkgRoot/usr/local/bin/"
+cp "${destDir}/runtime/start_tari_base_node.sh" "${destDir}/pkgRoot/usr/local/bin/runtime/"
```

Comment:
Is this FHS compliant? Are we willing to pollute install file system? Will this run without modifying the user path environment?

Reply:
A while back all the files for the different operating systems were aligned to use the same directory layout.

Suggestions?

Reply:
For archives, keeping the same folder layout is great, but for install packages I would suggest following the operating systems packaging suggests or running the risk of installs files/folders into poor locations. ie ```/usr/local/bin/runtime``` is not good. If you wish to install files and folders in a single directory, maybe ```/opt/tari_base_node```, but you will need admin access to write into ```/opt```

Reply:
Path has been updated to `/usr/local/tari` now.